pkg/api/grpc/daprinternal.go

```diff
@@ -281,34 +281,23 @@ func (a *api) CallLocalStream(stream internalv1pb.ServiceInvocation_CallLocalStr
 
 // CallActor invokes a virtual actor.
 func (a *api) CallActor(ctx context.Context, in *internalv1pb.InternalInvokeRequest) (*internalv1pb.InternalInvokeResponse, error) {
-	req, err := invokev1.InternalInvokeRequest(in)
-	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, messages.ErrInternalInvokeRequest, err.Error())
-	}
-	defer req.Close()
-
 	// We don't do resiliency here as it is handled in the API layer. See InvokeActor().
-	resp, err := a.Actors().Call(ctx, req)
+	res, err := a.Actors().Call(ctx, in)
 	if err != nil {
-		// We have to remove the error to keep the body, so callers must re-inspect for the header in the actual response.
 		actorErr, isActorErr := actorerrors.As(err)
-		if resp != nil && isActorErr {
-			defer resp.Close()
```

Comment:
Don't we want to keep this?

Reply:
No, as part of the refactoring this is now not a stream (since the data was never a stream to begin with for actor invocation), so no need to clsoe